sdc/tests/test_rolling.py

```diff
@@ -486,6 +486,24 @@ def test_impl(series, window, min_periods, center,
         msg = msg_tmpl.format('closed', 'int64', 'str')
         self.assertIn(msg, str(raises.exception))
 
+    @skip_sdc_jit('Series.rolling.count() unsupported Series index')
+    def test_series_rolling_count(self):
+        def test_impl(series, window, min_periods):
+            return series.rolling(window, min_periods).count()
+
+        hpat_func = self.jit(test_impl)
+
+        all_data = test_global_input_data_float64
+        indices = [list(range(len(data)))[::-1] for data in all_data]
+        for data, index in zip(all_data, indices):
+            series = pd.Series(data, index, name='A')
+            for window in range(len(series) + 2):
+                for min_periods in range(window + 1):
+                    with self.subTest(window=window, min_periods=min_periods):
```

Comment:
It might be useful to pass data and index into subTest too, to see the Series data on which the tests fails.